siguza-utils.mk

```diff
@@ -0,0 +1,83 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUPROJECTS      += siguza-utils
+# Don't change the version, just the date and git hash
+MISC_COMMIT     := 43852780a20ebf2c990681aa1ab42577ae630555
+MISC_VERSION    := 1.0+git20210410.$(shell echo $(MISC_COMMIT) | cut -c -7)
+DEB_MISC_V      ?= $(MISC_VERSION)
+
+siguza-utils-setup: setup
+	$(call GITHUB_ARCHIVE,Siguza,misc,$(MISC_COMMIT),$(MISC_COMMIT))
```

Comment:
Use arg 4 when changing the name of the tarball (look at ghostbin for reference)